operator/controllers/csiscaleoperator_controller.go

```diff
@@ -775,7 +775,7 @@ func (r *CSIScaleOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	//Do not restart driver pods when the configmap contains invalid Envs.
 	shouldRequeueOnCreateOrDelete := func(cfgmapData map[string]string) bool {
 		for key := range cfgmapData {
-			if strings.HasPrefix(strings.ToUpper(key), config.EnvVarPrefix) || strings.ToUpper(key) == config.DaemonSetUpgradeMaxUnavailableKey {
+			if strings.HasPrefix(strings.ToUpper(key), config.EnvVarPrefix) || strings.ToUpper(key) == config.DaemonSetUpgradeMaxUnavailableKey ||                                                                         strings.ToUpper(key) == config.HostNetworkKey{
```

Comment:
It would be a good idea to remove the extra spaces

Reply:
done